content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -6,5 +6,255 @@ weight: 1
 owner: istio/wg-networking-maintainers
 test: n/a
```

Comment:
I think this would be a good doc to test.
```suggestion
test: yes
```

Reply:
Agreed test would be good, but probably not part of this PR and can be a follow up PR, because we are trying to put the ambient upstream announcement blog out asap.